.eslintrc.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
```

Comment:
Alright, [check it out](https://github.com/uclaradio/uclaradio/wiki/Linters:-A-Quick-Introduction-on-Making-Your-Code-Pretty)!